extensions/encryption/middleware_after.go

```diff
@@ -4,16 +4,30 @@ import (
 	"github.com/s7techlab/cckit/router"
 )
 
-func EncryptInvokeResponse() router.MiddlewareFunc {
+func responseEncryptor(encryptionRequired bool) router.MiddlewareFunc {
 	return func(pre router.HandlerFunc, pos ...int) router.HandlerFunc {
 		return func(ctx router.Context) (interface{}, error) {
 			res, err := pre(ctx)
 
 			if err != nil || ctx.Handler().Type != router.MethodInvoke {
```

Comment:
here we can check presense of encode key in transient map and don't call EncryptWithTransientKey if there is not encode key and encryption not required

Reply:
fixed